@navikt/core/css/darkside/form/combobox.darkside.css

```diff
@@ -81,28 +85,25 @@
   cursor: text;
 }
 
-.navds-combobox--error .navds-text-field__input:not(:hover, :disabled) {
-  border-color: var(--ac-combobox-error-border, var(--a-border-danger));
-  box-shadow: 0 0 0 1px var(--ac-combobox-error-border, var(--a-border-danger));
+.navds-combobox__wrapper:has(.navds-combobox--error .navds-text-field__input:not(:hover, :disabled)) {
```

Comment:
Don't think this works as expected 
<img width="428" alt="Screenshot 2024-11-06 at 08 53 24" src="https://github.com/user-attachments/assets/c0217a3f-da81-4f3c-add9-dea93389f979">

We should avoid dynamically changing border-width, size this can cause layout-shifts